apps/test/unit/instructionsTest.js

```diff
@@ -15,117 +16,108 @@ describe('instructions', () => {
   testUtils.setExternalGlobals();
 
 describe('instructions reducer', () => {
-  var reducer = instructions;
+  const reducer = instructions;
 
 
   it('starts out uncollapsed', () => {
-    var state = reducer(undefined, {});
+    const state = reducer(undefined, {});
     assert.strictEqual(state.collapsed, false);
   });
 
   it('toggles collapsed', () => {
-    var initialState, newState;
+    let initialState, newState;
 
     // start collapsed
-    initialState = {
+    initialState = new InstructionsState({
       collapsed: false,
       longInstructions: 'foo'
```

Comment:
There is a sense in which our tests get more accurate too - where before we tested against an incomplete (and possibly invalid) state, now we know we're using a full record for every test.

Reply:
👍